anago

```diff
@@ -622,6 +622,21 @@ prepare_tree () {
   #            "Generating docs for ${RELEASE_VERSION[$label]} on $branch."
   #fi
 
+  # Clear all CHANGELOG-N.NN.md files on the branch.
+  # This replaces the above commented out change to the branch.
+  # Some unique change is required on the branch to allow it to be git
+  # describe'd as something other than the current tag on master
+  # If/when the generate-docs.sh issue is sorted out, this will become
+  # optional, though probably still useful anyway.
+  if [[ "$branch" =~ release- ]]; then
```

Comment:
It looks like the `$PARENT_BRANCH` check above was causing docs to be generated only at the initial branch point (beta.0), not for subsequent releases. Is that right? If so, is there a reason we wouldn't need that check here too?

Reply:
And this is why I keep sending you PRs. :)  Yes, that is better to keep that logic here as well.